plugin.html

```diff
@@ -1,34 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <title>Hello World</title>
-    <link href="plugin.css" rel="stylesheet">
-</head>
-<body>
-    <div class="pluginContainer">
-        <h1>Hello World</h1>
-        <p>Create a cuboid of the specified size at the origin.</p>
-        <div>
-            <input id="Width" type=number value =12 />
-            <label>Width</label>
-        </div>
-        <div>
-            <input id="Length" type=number value =12 />
-            <label>Length</label>
-        </div>
-        <div>
-            <input id="Height" type=number value =12 />
-            <label>Height</label>
-        </div>
-        <input id="CreateBlockBtn" type=button value="Create Cuboid" />
-        <!--Do not remove below scripts unless you know what you're doing-->
-        <script type="text/javascript" src="https://formit3d.github.io/SharedPluginUtilities/FormIt.js"></script>
```

Comment:
scripts are mostly kept outside the body content, though is just a minor suggestion

Reply:
I'll leave this for the moment as this seems to be the preference of the FormIt team